notebooks/06-cifar10-baseline.ipynb

```diff
@@ -0,0 +1,392 @@
+{
```

Comment:
<p>I'd suggest using <code>-qU</code>  instead of <code>--quiet</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyTorchLightning/pytorch-lightning/pull/4818/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>